pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -559,13 +558,6 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 	}
 	log.Printf("OpenShift API loadbalancer pool %s present", poolId)
 
-	log.Print("Creating OpenShift API loadbalancer health monitor")
-	monitorId, err := ensureOpenStackLbMonitor(lbClient, "kuryr-api-loadbalancer-monitor", poolId)
```

Comment:
I have concerns myself by removing that function and it makes me wonder if we should keep this around in case OVN provider isn't detected as available?
There is an assumption that OVN backend is enabled since the recent versions of OSP, but I don't think it makes it a safe assumption either a backward compatible change.